package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "minicart",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "lint": "eslint --ext js,jsx,ts,tsx ."
```

Comment:
Just to make sure, did you run `yarn lint` before your last push?
I ask that because I did not include `js, jsx` files to be linted since they are from the legacy minicart.
If the lint passes, then it's fine :)

Reply:
the lint isn't passing, but I think I'll add `react/legacy` to `.eslintignore`, so I won't need to fix all issues on the legacy code right now